src/lib/Scenes/Home/Components/ForYou/Components/FairsRail.tsx

```diff
@@ -58,7 +58,7 @@ export class FairsRail extends Component<Props, null> {
       }
 
       const selectionHandler = () => {
-        Switchboard.presentNavigationViewController(this, `${fair.profile.href}?entity=fair`)
+        Switchboard.presentNavigationViewController(this, `${fair.id}?entity=fair`)
```

Comment:
I just did something like this - doesn't it need to be different depending on whether it has a profile or not? I just had a bug like this in native code

Reply:
https://github.com/artsy/eigen/pull/2772